nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -1392,6 +1404,33 @@ public boolean accept(final File dir, final String filename) {
         }
     }
 
+    private String printBasicOSJavaDetails(final String xms, final String xmx) {
+        Map<String, String> details = new HashMap<String, String>(6);
```

Comment:
Recommend `LinkedHashMap` for guaranteed ordering.
```suggestion
        Map<String, String> details = new LinkedHashMap<String, String>(6);
```